svy/src/utils/tokens.ts

```diff
@@ -0,0 +1,11 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+/**
+ * Convert svyBalance in USD value.
+ * @param svyBalance The balance of SVY token.
+ * @returns The USD value of svyBalance.
+ */
+export function getSvyBalanceInUSD(svyBalance: BigInt): BigDecimal {
+  const svyPriceInUSD = BigDecimal.fromString("3");
+  return svyBalance.toBigDecimal().times(svyPriceInUSD);
```

Comment:
Set the price in natural USD (with decimals accounted for).

```suggestion
  return svyBalance.toBigDecimal().times(svyPriceInUSD).div(BIGDECIMAL_TEN_TO_EIGHTEENTH);
```